packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -1004,7 +1004,7 @@ function useMutableSource<Source, Snapshot>(
         // There is no mechanism currently to associate these updates though,
         // so for now we fall back to synchronously flushing all pending updates.
         // TODO: This should entangle the lanes instead of expiring everything.
```

Comment:
This TODO is now TODONE, right? 😉

Also the block comment above is kind of stale.

Reply:
I'll rewrite :)